packages/arb-token-bridge-ui/src/components/common/NetworkSelectionContainer.tsx

```diff
@@ -48,36 +48,38 @@ export const NetworkSelectionContainer = ({
       <Transition>
         <Popover.Panel className="relative flex flex-col rounded-md lg:absolute lg:ml-1 lg:mt-1 lg:bg-white lg:shadow-[0px_4px_20px_rgba(0,0,0,0.2)]">
           {({ close }) =>
-            supportedNetworks?.map((chainId, i) => (
-              <div // TODO: replace with button
-                key={chainId}
-                className="flex h-12 cursor-pointer flex-nowrap items-center justify-start space-x-3 px-12 text-lg font-light text-white hover:bg-[rgba(0,0,0,0.2)] lg:px-4 lg:text-base lg:font-normal lg:text-dark"
-                onClick={() => {
-                  handleClick(chainId, close)
-                }}
-                onKeyDown={e => {
-                  if (e.key === 'Enter' || e.keyCode === 13) {
+            supportedNetworks
+              ?.filter(chainId => chainId !== chain?.id)
```

Comment:
A change on this line generates an unnecessarily large JSX diff, so it might be cleaner to just do it on line 24 like this:

```
const supportedNetworks = getSupportedNetworks(chain?.id)
  // filter out the current network
  ?.filter(chainId => chainId !== chain?.id)
```

Reply:
Thank you for reviewing my code, I've pushed the changes on the new commit.